include/cpr/error.h

```diff
@@ -89,6 +90,71 @@ enum class ErrorCode {
     UNKNOWN_ERROR = 1000,
 };
 
+inline std::unordered_map<ErrorCode, std::string> error_code_to_string_mapping = {{ErrorCode::OK, "OK"},
```

Comment:
```suggestion
inline const std::unordered_map<ErrorCode, std::string> error_code_to_string_mapping = {{ErrorCode::OK, "OK"},
```

That's my only suggestion to prevent others from modifying it.

Other than that well done. Thanks for pushing it past the last mile!

Reply:
Done. 👍